flit/wheel.py

```diff
@@ -45,11 +47,21 @@ def clean_build_dir(self):
             self.build_dir.mkdir()
 
     def copy_module(self):
+
+        def verbose_copy2(src, dest, *args, **kwargs):
+            log.debug('copying %s', src)
+            return copy2(src, dest, *args, **kwargs)
+
+
         mod =  self.module
         # Copy module/package to build directory
         if mod.is_package:
             ignore = shutil.ignore_patterns('*.pyc', '__pycache__')
-            shutil.copytree(str(mod.path), str(self.build_dir / mod.name), ignore=ignore)
+            shutil.copytree(str(mod.path), str(self.build_dir / mod.name),
+                    ignore=ignore,
+                    symlinks=True,
```

Comment:
Is there any use in setting this? It's either going to fail later or lose the symlink relationships when it creates the zip file, right?